### PR TITLE
remove unused tracker files, clean up console and filenames

### LINKED ISSUE #146: Strip out the RNG Tracker GUI

### LINKED ISSUE #160: Rename zz_eggHuntAuto.py
Having the zz prefix no longer appropriate for this. This file does the controls for the egg hunt, so if there's an appropriate way to isolate this file and the functionality therein, we can treat the egg hunt as its own isolated function.

### PR DESCRIPTION
**Pull request by shenef**
resolves #160
resolves #146

### CODE REVIEW

**Comment by shenef** [comment]
changes have been verified on a full any% run

File: log_init.py [hunk 1]
```diff
@@ -127,17 +127,14 @@ def _add_log_level(level_name: str, level_num: int, method_name: Optional[str] =
     """
     Comprehensively adds a new logging level to the `logging` module and the
     currently configured logging class.
-
```
Comment (jlward) [inline log_init.py:130]: I feel like these make sense to keep. They establish a break in thoughts in the docstring.

File: tracker/data/notes/anypercent/encounters_notes.csv [hunk 2]
```diff
@@ -1,117 +0,0 @@
-#name,type,initiative,label,min,default,max
```
Comment (jlward) [inline tracker/data/notes/anypercent/encounters_notes.csv:1]: Do we know why all these files were included to begin with? I'm hesitant to check-mark removing 100 files without at least a little bit of context.

File: log_init.py (hunk 1 above)
Comment (shenef) [reply log_init.py:130]: Oh i agree, that probably was the black formatter removing it. I'll check on that

File: tracker/data/notes/anypercent/encounters_notes.csv (hunk 2 above)
Comment (shenef) [reply tracker/data/notes/anypercent/encounters_notes.csv:1]: They were included cause the whole RNG Tracker repository that the RTA community uses was just dropped into the "tracker" folder. That includes all of its UI code and the default tracking notes RTA runners use when working with the UI.

**Review by coderwilson — CHANGES_REQUESTED** [review]
@shenef I'd like the file tracker/ffx_rng_tracker/events/yojimbo_turn.py to remain, and then anything that makes it work. If we can leverage this one file, it will save gil for the run and potentially help us land under-gil runs.

File: tracker/data/notes/anypercent/encounters_notes.csv (hunk 2 above)
Comment (coderwilson) [reply tracker/data/notes/anypercent/encounters_notes.csv:1]: Can confirm, I just copied the whole thing and treated it as a library for the parts needed within the TAS.

**Review by coderwilson — APPROVED** [review]
Looks good.

**Pull request merged**